src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1955,9 +1954,20 @@ void Session::send_bind_message()
     int protocol_version = m_conn.get_negotiated_protocol_version();
     OutputBuffer& out = m_conn.get_output_buffer();
     // Discard the token since it's ignored by the server.
-    std::string empty_access_token{};
-    protocol.make_bind_message(protocol_version, out, session_ident, path, empty_access_token, need_client_file_ident,
-                               is_subserver); // Throws
+    std::string empty_access_token;
+    if (m_is_flx_sync_session) {
+        nlohmann::json bind_json_data;
+        if (auto migrated_partition = get_migration_store()->get_migrated_partition()) {
+            bind_json_data["partitionKey"] = *migrated_partition;
```

Comment:
The server is expecting the key to be `migratedPartition`

Reply:
Thanks - I forgot to update the key name